source/GlassView.Export/Globals.cs

```diff
@@ -0,0 +1,3 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("GlassView.Export.Test")]
```

Comment:
Isn't this already specified in the csproj?

Reply:
Thanks. Deleted the file :-)